### PR TITLE
Added new Admin setting dashboard page

### PR DESCRIPTION
**Pull request by vidhya9lakshmi**

### CODE REVIEW

File: src/app/admin_modules/admin_dashboard/admin-dashboard-modules.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { FormsModule } from '@angular/forms';
+import { UserDataService } from './services/user-data.service';
+import { HttpClientModule } from '@angular/common/http';
+import { AdminFilterPipe } from './pipes/filter.pipe';
+import { OrderByPipe } from './pipes/order-by.pipe';
+import { DashEditComponent } from './dashboard/admin-dashboard/dash-edit/dash-edit.component';
+import { ManageAdminsComponent } from './dashboard/admin-dashboard/manage-admins/manage-admins.component';
+import { EditTokenModalComponent } from './dashboard/admin-dashboard/modal/edit-token-modal/edit-token-modal.component';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+// tslint:disable-next-line:max-line-length
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/admin-dashboard-modules.ts:15]: @vidhya9lakshmi  remove commented code

File: src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/admin-dashboard.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Component({
+  selector: 'app-admin-dashboard',
+  templateUrl: './admin-dashboard.component.html',
+  styleUrls: ['./admin-dashboard.component.scss']
+})
+export class AdminDashboardComponent implements OnInit {
+  tab = 'dashboards';
+  constructor() { }
+
+  ngOnInit() {
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/admin-dashboard.component.ts:12]: @vidhya9lakshmi  onInit , can we default to editDashboards?

File: src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/generate-api-tokens/generate-api-tokens.component.scss [hunk 3]
```diff
@@ -0,0 +1,349 @@
+// WIDGET
+// ========================================
+.widget {
+    padding: 2.5rem 3rem;
+   // .border-radius(@dash-widget-border-radius);
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/generate-api-tokens/generate-api-tokens.component.scss:5]: remove commented code

File: src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/manage-admins/manage-admins.component.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  loadUser() {
+    this.userData.users().subscribe((response: any) => {
+      this.users = response;
+      console.log('users ', response);
+    });
+  }
+  promoteUserToAdmin(user) {
+
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/manage-admins/manage-admins.component.ts:28]: is this functionality work in progress

File: src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/modal/delete-confirm-modal/delete-confirm-modal.component.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      <button type="button" class="close" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+    <div class="modal-body">
+      {{message}}
+    </div>
+    <div class="modal-footer text-center">
+      <button type="button" class="btn btn-secondary btn-wide"
+        (click)="activeModal.dismiss('dismiss')">{{bntName1}}</button>
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/modal/delete-confirm-modal/delete-confirm-modal.component.html:14]: add valid naming convention for btnName1 and btnName2

File: src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/modal/delete-confirm-modal/delete-confirm-modal.component.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'app-delete-confirm-modal',
+  templateUrl: './delete-confirm-modal.component.html',
+  styleUrls: ['./delete-confirm-modal.component.scss']
+})
+export class DeleteConfirmModalComponent implements OnInit {
+
+  public title = '';
```
Comment (nireeshT) [inline src/app/admin_modules/admin_dashboard/dashboard/admin-dashboard/modal/delete-confirm-modal/delete-confirm-modal.component.ts:11]: title required ?

File: src/app/admin_modules/admin_dashboard/admin-dashboard-modules.ts (hunk 1 above)
Comment (vidhya9lakshmi) [reply src/app/admin_modules/admin_dashboard/admin-dashboard-modules.ts:15]: Exceeds maximum line length of 140 (max-line-length) for import

**Pull request merged**